Nodejs/Product/Nodejs/TestFrameworks/Tape/tape.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+var fs = require('fs');
+var path = require('path');
+
+function find_tests(testFileList, discoverResultFile, projectFolder) {
+    var test = findTape(projectFolder);
+    if (test === null) return;
```

Comment:
nit: siorry for not defining this more clearly, but just to be consistent with the rest of our codebase... if statements should use braces and no single-line ifs